tutorials/local-server-api-db-intro.md

```diff
@@ -0,0 +1,71 @@
+# Local Server
+
+## Overview 
+
+In this module we will bring up the API server and do some basic exploration of the database and make an API call against the server. The goal is to help you gain an understanding of how the back end and data is structured and to start gaining an understanding the tools, scripts and languages we use to aid us in development.
+
+## Tasks
+
+### (LOCAL-SERVER-1) Bring up the local server
+
+./project.rb dev-up
+
+This will bring up a server with API endpoints and a local database.
+
+
+#### Dependencies
+You should have the following installed on your machine for this to work
+
+- Ruby installed
+- docker installed and running
+- docker-compose
+- docker-sync (`gem install docker-sync` or `gem install --user-install docker-sync` or non-root installs)
+
+Having ruby gems installed on your path is a good idea as well - adding this to your profile or rc file will help to find the ruby gems:
+
+```sh
+if which ruby >/dev/null && which gem >/dev/null; then
+  PATH="$(ruby -r rubygems -e 'puts Gem.user_dir')/bin:$PATH"
+fi
+```
+
+### (LOCAL-SERVER-2) Using a REST client (e.g. insomnia) make a request to the local server
+  1. Start with the /v1/profile endpoint (http://localhost:8081/v1/profile)
+  2. What was the response?
```

Comment:
nit: TBH, I'd probably assume a bug in the tutorial or that I messed up my local server when I get a 401 here. May want a stronger hint, like "What error response do you get? Why?"